db/seed.js

```diff
@@ -0,0 +1,89 @@
+const { saveReview } = require('./db');
+const faker = require('faker');
+const axios = require('axios');
+
+const seedDb = ( bizCount = 100 ) => {
+  let hipsterReviews = [];
+  let meatReviews;
+  let starReviews = [];
+  // Get Hipser Reviews
+  axios.get('http://hipsterjesus.com/api/')
+  .then(({data}) => {
+    // Split paragraph into variable lengths
+    let hipsterParagraph = data.text.split(' ');
+    let start = 0;
+    let end;
+    while ( start < hipsterParagraph.length) {
+      end = getRandom(start, 5, 8);
+      let curr = hipsterParagraph.slice(start, end).join(' ') + '.';
+      curr = curr.charAt(0).toUpperCase() + curr.slice(1);
+      hipsterReviews.push(curr);
+      start = end;
+    }
+    // Get Bacon Ipsum Reviews
+    axios.get('https://baconipsum.com/api/?type=all-meat&paras=25&start-with-lorem=1')
+    .then(({data}) => {
+      meatReviews = data;
+      // Get Reviews by Star Rating
+      let args = [];
+      for (let star = 1; star <= 5; star++) {
+        for(let i = 0; i <= 7; i++) {
+          args.push( axios.post(`http://reviews.azurewebsites.net/?stars=${star}`) );
+        }
+      }
+      axios.all(args)
+      .then(axios.spread( (...allReviews) => {
```

Comment:
It's good to learn about how to use promise for axios here. 😄 
I believe async/await will make your seeding process more straightforward and easier to read.

Reply:
I'll definitely have to look into how you implemented it. I spent an hour attempting, though it ultimately didn't work successfully.